### PR TITLE
morewaita-icon-theme: 47.1-> 47.2

### PR DESCRIPTION
**Pull request by nbCloud91**
**Summary:**

This pull request updates the `morewaita` package from version 47.1 to 47.2.

---
## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#linking-nixos-module-tests-to-a-package) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [25.05 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2505.section.md) (or backporting [24.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) and [25.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2505.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

**Comment by getchoo** [comment]
@ofborg build morewaita-icon-theme

File: pkgs/by-name/mo/morewaita-icon-theme/package.nix [hunk 1]
```diff
@@ -7,13 +7,13 @@
 }:
 stdenvNoCC.mkDerivation rec {
   pname = "morewaita-icon-theme";
-  version = "47.1";
+  version = "47.2";
 
   src = fetchFromGitHub {
     owner = "somepaulo";
     repo = "MoreWaita";
     rev = "${version}";
```
Comment (getchoo) [inline pkgs/by-name/mo/morewaita-icon-theme/package.nix:15]:
```suggestion
    rev = "refs/tags/v${version}";
```
Seems that upstream has started prefixing their tags with `v`, so we need to reflect that here in order to get the right URL

**Review by getchoo — CHANGES_REQUESTED** [review]
And welcome to Nixpkgs!

**Comment by getchoo** [comment]
@ofborg build morewaita-icon-theme

**Review by getchoo — APPROVED** [review]
Feel free to ping me in a couple days if there are no other reviews :)

**Review by Aleksanaa — APPROVED** [review]

**Comment by Aleksanaa** [comment]
Tips: Add

```
Diff: https://github.com/somepaulo/MoreWaita/compare/47.1...v47.2
Changelog: https://github.com/somepaulo/MoreWaita/releases/tag/v47.2
```

to commit message to ease reviewing updates. I'm adding for you this time.

**Pull request merged**